.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: ruby
+rvm:
+ - 2.2
+ - jruby
```

Comment:
i don't think we need to test against mutlitple ruby versions.  I would say `2.2` is enough